src/main/java/gim/logic/commands/GenerateCommand.java

```diff
@@ -28,47 +29,65 @@ public class GenerateCommand extends Command {
     public static final String COMMAND_WORD = ":gen";
     public static final String DIFFICULTY_LEVELS = "{easy, medium, hard}";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD
-            + " -> Generates a sample workout based on PRs (Personal Records) "
-            + "of the exercise(s) identified by the indices number used in the displayed exercise list, "
-            + "according to the difficulty level specified. " + "\n"
-            + "Difficulty levels supported are: " + DIFFICULTY_LEVELS + "\n"
-            + "Parameters: INDEX_1 [, INDEX_2, ...] (each index must be a positive integer) "
-            + PREFIX_LEVEL + "LEVEL (must be one of " + DIFFICULTY_LEVELS + ") " + "\n"
-            + "Example usage: " + COMMAND_WORD + " 2,3 " + PREFIX_LEVEL + "easy";
+    public static final String MESSAGE_USAGE = "The generate feature has two variations: \n\n"
+            + COMMAND_WORD + " INDEX [, INDEX]... " + PREFIX_LEVEL + "LEVEL\n"
```

Comment:
To standardise with the UG, perhaps you could remove the comma before the optional index

Reply:
I think we should keep this, comma is used as the separator for the indexes